causalml/inference/meta/rlearner.py

```diff
@@ -271,6 +299,56 @@ def bootstrap(self, X, p, treatment, y, size=10000):
         te_b = self.predict(X=X)
         return te_b
 
+    def compute_propensity_score(self, X, treatment, X_pred=None, treatment_pred=None, cv=None, calibrate_p=True):
```

Comment:
This function doesn't access anything in the class instance. Let's move it to `propensity.py` and import from it. It will also prevent from repeating this code in `xlearner.py` as well.

Reply:
Made the change. Yeah, this is a great suggestion and will make the code simpler. Thanks!